### PR TITLE
template.txt

### PR DESCRIPTION
**Pull request by trraney328**

### CODE REVIEW

**Comment by CyberHacker101** [comment]
oof i had dinner

**Comment by trraney328** [comment]
its okay

**Comment by trraney328** [comment]
accept my review request invitation

**Comment by CyberHacker101** [comment]
ok

File: template.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  else:
+    #Task 9
+    #Minus one value from a variable here
+    life -= 1
+    #append input_letter into chosen_letters here
+    chosen_letters.append(input_letter) 
+    print(display_blanks())
+
+print("Game over!")
+print("Answer:", answer)
```
Comment (CyberHacker101) [inline template.txt:63]: Idk what to do but ill figure it out

**Comment by trraney328** [comment]
hey
review my work, okay?

**Review by CyberHacker101 — COMMENTED** [review]
Ummmm ill do this later k?

**Comment by trraney328** [comment]
hhh, okay but make sure that you review my work, okay?

File: template.txt (hunk 1 above)
Comment (trraney328) [reply template.txt:63]: you'll see that box

**Comment by CyberHacker101** [comment]
k I'm just a little busy these days

**Comment by trraney328** [comment]
I know

**Comment by CyberHacker101** [comment]
DUDE STAY ON ONE THING

**Comment by trraney328** [comment]
okay

**Comment by trraney328** [comment]
hey, what did you do?

**Comment by trraney328** [comment]
hey

**Comment by trraney328** [comment]
@CyberHacker101

**Comment by CyberHacker101** [comment]
WHAT DO U WANT

**Comment by trraney328** [comment]
hey, can you review this for me?

**Comment by trraney328** [comment]
and calm yourself down

**Comment by CyberHacker101** [comment]
IM IN MATHHH MAN I GOTTA GOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by trraney328** [comment]
O_O woops sorry

**Comment by trraney328** [comment]
hey let me know when your done with your math okay?
bye!

**Comment by CyberHacker101** [comment]
I KEEP TYPING IN CAPS BECAUSE MY KEYBOARD IS A WEIRDO

**Comment by trraney328** [comment]
well, that happens though

**Comment by CyberHacker101** [comment]
YEA BREAK STARTS AT 11:30 CYA HERE LATER

**Comment by trraney328** [comment]
but
when you get done with your math let me know in the discussion and i'll get the ready for review
okay?

**Comment by trraney328** [comment]
bye

**Comment by CyberHacker101** [comment]
K

**Comment by trraney328** [comment]
try turning your caps off

**Comment by trraney328** [comment]
CYA

**Comment by trraney328** [comment]
and don't forget to accept my review invitation

**Comment by trraney328** [comment]
hhhh

**Comment by trraney328** [comment]
hey

**Comment by trraney328** [comment]
@CyberHacker101

**Comment by CyberHacker101** [comment]
oof sorry I am very busy Idk if I got time to code

**Comment by trraney328** [comment]
well to review it though cyberhacker101 you just need to review it

**Comment by trraney328** [comment]
so, are you ready to review this?

**Comment by trraney328** [comment]
please?

**Comment by trraney328** [comment]
what just happened?

**Comment by trraney328** [comment]
oh nvm

**Comment by trraney328** [comment]
hey @CyberHacker101 are you done yet?

**Comment by CyberHacker101** [comment]
dude id think I got time to code this week

File: template.txt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      display_txt += letter + " "
+    else:
+      display_txt += "_ "
+  
+  #Task 5 - return the complete display_txt
+  return display_txt + "\n"
+  
+#print the blank spaces based on the length of
+#the answer when the game starts  
+print("_ " * len(answer))
```
Comment (CyberHacker101) [inline template.txt:29]: It better if the word is separated from each other like Christmastable just add the space or make it a new word such as remove the table part

**Review by CyberHacker101 — COMMENTED** [review]
U Better look at these comments man there good advice :D

File: template.txt (hunk 2 above)
Comment (trraney328) [reply template.txt:29]: okay

**Pull request closed without merging**